README.md

```diff
@@ -122,3 +122,7 @@ Credits for Images:
 * MatureBlackberryBush: Made by VaibhavBajaj (inspired by Testberry)
 * BlackberryBushFull: Made by VaibhavBajaj (inspired by Testberry)
 * Cranberry and all related: [András Ottó Földes](https://github.com/andriii25)
+* Orange: http://www.freestockphotos.biz/stockphoto/11403 (Resized)
+* OrangeSeed: https://pixabay.com/en/seeds-shells-yellow-foods-nuts-576562/ (Resized)
+* OrangeBush : http://maxpixel.freegreatpicture.com/Tree-Png-Time-Of-Year-Leaves-Isolated-Autumn-1763719 (Resized)
```

Comment:
Some of these images are huuuuuge. With them resized down that seems to have resulted in weirdly blurry and somewhat translucent images. You might want to find alternatives or just make some low resolution bush textures yourself :-)

They also need to be square. One of them isn't and that block doesn't actually work.

Additionally they end up looking problematic in-game - will attach screenies.